docs/EN_US/HPCCSpark/SparkHPCC.xml

```diff
@@ -276,19 +276,19 @@
                   </row>
 
                   <row>
-                    <entry>SPARK_MASTER_WEBUI_PORT</entry>
+                    <entry>SPARK_MANAGER_WEBUI_PORT</entry>
```

Comment:
Is this a Spark managed variable? @Michael-Gardner do we control this?

Reply:
@g-pan this should be reverted back to SPARK_MASTER_WEBUI_PORT.  It's not a variable that we control on our side of things so it has to keep the older language for the actual variable. But I'd keep the change on the description that calls it  manager instead of master.

Reply:
done